cfme/web_ui/__init__.py

```diff
@@ -1280,6 +1280,7 @@ def _fill_form_list(form, values, action=None, action_always=False):
 
     """
     logger.info('Beginning to fill in form...')
+    sel.wait_for_ajax()
```

Comment:
After the form is loaded....do you mean when navigating to the page? Or after selecting a provider?